user_guide/system_config.rst

```diff
@@ -231,7 +231,25 @@ Sidebar settings
 With the Sidebar settings you can enable or disable the left and/or right sidebar to keep your Sticky notes and Task lists. 
 By default only the right sidebar is enabled.
 
+Reports settings
+^^^^^^^^^^^^^^^^
+
+If the function is enabled, the users can see the SQL request sent to the system for a report. 
```

Comment:
Edit: If this function is enabled, users can see the SQL request sent to the system for a report.